cvmfs/hash.h

```diff
@@ -54,7 +57,7 @@ const char kSuffixCertificate  = 'X';
  * Corresponds to Algorithms.  "Any" is the maximum of all the other
  * digest sizes.
  */
-const unsigned kDigestSizes[] = {16, 20, 20, 32, 32};
+const unsigned kDigestSizes[] = {16, 20, 20, 32, 32, 20, 32};
```

Comment:
I have to feeling that this gets a bit out of hand. With seven supported hash algorithms I think this starts to reach the boundary of unsupportability. Hard to say on the spot what would be a better solution, I'll have a look tomorrow.